regions/io/stcs/README.md

```diff
@@ -0,0 +1,233 @@
+
+# Position: lon lat
+Position FK5 GEOCENTER 85.0 -15.0
+
+# Pixel coordinates
+Circle IMAGE UNKNOWN 100.5 200.3 15.0
+```
+
+### Coordinate Frames
+
+- **ICRS**: International Celestial Reference System
+- **FK5**: Fifth Fundamental Catalogue (J2000.0)
+- **FK4**: Fourth Fundamental Catalogue (B1950.0)
+- **GALACTIC**: Galactic coordinate system
+- **ECLIPTIC**: Ecliptic coordinate system
+- **IMAGE**: Pixel/image coordinates
+
+### Reference Positions
+
+- **BARYCENTER**: Solar system barycenter
+- **GEOCENTER**: Earth center
+- **TOPOCENTER**: Earth surface/topocentric
+- **HELIOCENTER**: Sun center
+- **LSR**: Local Standard of Rest
+- **UNKNOWN**: Unspecified reference
+
+## File Format
+
+STC-S files typically have extensions:
+- `.stcs`
+- `.stc`
+- `.stcs.txt`
+- `.stc.txt`
+
+Files can contain:
+- Comments starting with `#`
+- Multiple regions, one per line
+- Blank lines (ignored)
+
+Example file:
+```stcs
+# STC-S region file
+# Generated by astropy-regions
+
+# Central source
+Circle ICRS BARYCENTER 180.0 10.0 0.5
+
+# Extended emission
+Ellipse ICRS BARYCENTER 150.0 -20.0 1.0 0.5 45.0
+
+# Point sources
+Position FK5 GEOCENTER 85.0 -15.0
+Position FK5 GEOCENTER 90.0 -10.0
+```
+
+## Implementation Details
+
+The STC-S module consists of:
+
+- **`core.py`**: Core parsing functions, mappings, and utilities
+- **`connect.py`**: File format identification and registry
+- **`read.py`**: STC-S reading and parsing functionality
+- **`write.py`**: STC-S writing and serialization functionality
+- **`tests/`**: Comprehensive test suite
+
+### Key Functions
+
+- `validate_stcs_string()`: Validate STC-S format
+- `parse_coordinate_frame()`: Extract coordinate frame and reference position
+- `parse_numbers()`: Parse numeric parameters
+- `_parse_stcs()`: Main parsing function
+- `_serialize_stcs()`: Main serialization function
+
+## References
+
+- [IVOA STC-S Standard](https://www.ivoa.net/documents/Notes/STC-S/20091030/NOTE-STC-S-1.33-20091030.html)
+- [CDS STC-S Rust Implementation](https://github.com/cds-astro/cds-stc-rust/)
+- [astropy-regions Issue #21](https://github.com/astropy/regions/issues/21)
+
+## Testing
+
+Run the test suite:
+```bash
+# Run all STC-S tests
+pytest regions/io/stcs/tests/
+
+# Run specific test file
+pytest regions/io/stcs/tests/test_stcs.py
+
+# Run with verbose output
+pytest regions/io/stcs/tests/ -v
+```
+
+Test data files are located in `regions/io/stcs/tests/data/`.
+
+## Contributing
+
+When contributing to the STC-S module:
+
+1. Follow the existing code style and patterns from the DS9 module
+2. Add comprehensive tests for new functionality
+3. Update documentation and examples
+4. Ensure round-trip conversions work correctly
+5. Test with various coordinate systems and shapes
+
+## Known Limitations
+
+- Complex STC-S features like time coordinates are not yet supported
+- Union, intersection, and other compound operations are not implemented
```

Comment:
seems reasonable - does the standard even support them?

Reply:
yes: "Optionally, compound sub-phrases may be recognized for compound Regions. Under those rules the CoordinateArea operational identifier (Union, Intersection, or Difference) precedes the CoordinateFrame component which, in turn, is followed by two or more (if applicable) CoordinateArea components; these CoordinateArea identifiers may be preceded by the negation operator Not. The arguments for these operators must be enclosed in parentheses. For Union and Intersection the argument may contain two or more elements; for Difference it must contain exactly two; and for Not it must contain only one. The enclosure in parentheses is merely intended to allow nesting and avoid ambiguities."